buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateIntellijVersions.kt

```diff
@@ -0,0 +1,105 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.getEapOfLatestSupportedMajor
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.getMajorPart
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.toBuildNumber
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.versionIsNewerThan
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import org.jsoup.Jsoup
+import org.jsoup.select.Elements
+
+open class UpdateIntellijVersions : DefaultTask() {
+
+  private val intellijReleasesContents: Elements by lazy {
+    getLinksFromUrl("https://www.jetbrains.com/intellij-repository/releases/")
+  }
+
+  private val intellijSnapshotsContents: Elements by lazy {
+    getLinksFromUrl("https://www.jetbrains.com/intellij-repository/snapshots/")
+  }
+
+  private fun getLinksFromUrl(repositoryUrl: String): Elements {
```

Comment:
Instead of returning HTML elements here, pls limit the results to just the URLs themselves (since that's what's really relevant); can be just a `List<String>` or whatnot